docs/design_doc.md

```diff
@@ -202,44 +193,29 @@ Regions can be defined in multiple ways.
 
 ##### Labels (pixel mask)
 
-Labels are a pixel mask representation of regions. This is an array of integers where each integer value corresponds to a region of space. This is commonly used along side pixel based imaging techniques, where the label array will share dimensionality with the image array. These may also be hierarchichal.
+Labels are a pixel mask representation of regions. This is an array of integers where each integer value corresponds to a region of space. This is commonly used along side pixel-based imaging techniques, where the label array will share dimensionality with the image array. These may also be hierarchichal.
 Should conform to the [OME-NGFF definition](https://ngff.openmicroscopy.org/latest/#image-layout).
 
 The Python data structures used for Labels are the same one that we discussed for Images.
 
 The coordinate systems and transforms are stored in `spatial_image.SpatialImage.attrs` or in `multiscale_spatial_image.MultiscaleSpatialImage.attrs`.
 
+We require the following axes (in the following order):
+
+-   2D labels: yx
+-   3D labels: zyx
+
 ##### Shapes
 
 A set of (multi-)polygons or points (circles) associated with a set of observations. Each set of polygons is associated with a coordinate system. Shapes can be used to represent a variety of regions of interests, such as clinical annotations and user-defined regions of interest. Shapes can also be used to represent most of array-based spatial omics technologies such as 10X Genomics Visium, BGI Stereo-seq and DBiT-seq.
 
 The Shapes object is implemented as a geopandas dataframe with its associated [geopandas data structure](https://geopandas.org/en/stable/docs/user_guide/data_structures.html). The coordinate systems and transforms are stored in `geopandas.DataFrame.attrs`.
 We are considering using the [dask-geopandas library](https://dask-geopandas.readthedocs.io/en/stable/), [discussion here](https://github.com/scverse/spatialdata/issues/122).
 
-**Warning**
```

Comment:
Removed because some parts are out of date (like when we say that shapes are saved as AnnData objects), and because the problem of storing the centroids of labels in a table will disappear with the new transformations refactoring (because cropping will leave the xarray coordinates unaltered and the xarray object will **live** in a coordinate system, as opposed of now that is **mapped** to a coordinate system). Until the transformation refactoring is done the problem is still open, but deleting this from the design doc anyway because we don't use the centroids of labels anywhere and we are not likely going to implement anything based on it before the transformation refactoring is done.